ansible/roles/test/tasks/link_flap_fib.yml

```diff
@@ -0,0 +1,56 @@
+#------------------------#
+# Run Link Flap FIB Test #
+#------------------------#
+
+# Pre-check testbed_type value
+- fail: msg="testbed_type is not defined."
+  when: testbed_type is not defined
+
+- fail: msg="testbed_type {{testbed_type}} is invalid."
+  when: testbed_type not in ['t1-lag', 't1', 't0']
+
+- include_vars: "vars/topo_{{testbed_type}}.yml"
+
+- name: Expand properties into props
+  set_fact: props="{{configuration_properties['spine']}}"
+  when: testbed_type in ['t1', 't1-lag']
+
+- name: Expand properties into props
+  set_fact: props="{{configuration_properties['common']}}"
+  when: testbed_type in ['t0']
+
+- name: Expand ToR properties into props
+  set_fact: props_tor="{{configuration_properties['tor']}}"
+  when: testbed_type in ['t1', 't1-lag']
+
+- name: Gathering minigraph facts about the device
+  minigraph_facts: host={{ inventory_hostname }}
+  connection: local
+
+# Generate route file
+- name: Generate route-port map information
+  template: src=roles/test/templates/fib.j2
```

Comment:
you can use template to deploy the file to remote host, it can be combine with copy. not sure if you need to separate them.